transfluent.py

```diff
@@ -117,7 +117,7 @@ def texts_save(self, group_id, language, texts,
             'language': language,
             'invalidate_translations': 1 if invalidate_translations else 0,
         }
-        for key, content in texts.iteritems():
+        for key, content in texts.items():
```

Comment:
Actually, since we don't have a package here, but only `transfluent.py` just add the compatibility stuff on top of `transfluent.py`.